datafusion/src/physical_plan/planner.rs

```diff
@@ -76,15 +75,25 @@ fn create_function_physical_name(
     Ok(format!("{}({}{})", fun, distinct_str, names.join(",")))
 }
 
-fn physical_name(e: &Expr, input_schema: &DFSchema) -> Result<String> {
+fn physical_name(e: &Expr) -> Result<String> {
```

Comment:
nitpick, by just looking at the function names, it's not easy to tell the difference between `physical_name` and `create_physical_name`. Perhaps rename `physical_name` to `root_physical_name`?

Alternatively, we could get rid of the wrapper, then use `physical_name` and explicitly pass in the `is_first_expr` argument everywhere.

Reply:
It's a good point to remove the wrapper. `physical_name()` is an interior helper function and adding that argument won't be too verbose.